Feature - Add nodeSelector, affinity and tolerations to daemonset and deployment templates (#456)
nginxinc/kubernetes-ingress: deployments/helm-chart/README.md

`controller.defaultTLS.secret` | The secret with a TLS certificate and key for the default HTTPS server. The value must follow the following format: `<namespace>/<name>`. Used as an alternative to specifiying a certifcate and key using `controller.defaultTLS.cert` and `controller.defaultTLS.key` parameters. | None
`controller.nodeSelector` | The node selector for pod assignment for the Ingress controller pods. | { }
`controller.terminationGracePeriodSeconds` | The termination grace period of the Ingress controller pod. | 30

> `controller.tolerations` | The taints to tolerate for the Ingress controller pods. | []

Reviewer: ```suggestion
`controller.tolerations` | The tolerations of the Ingress controller pods. | []
```

> `controller.affinity` | The affinities of the ingress controller pods. | { }

Reviewer: ```suggestion
`controller.affinity` | The affinity of the Ingress controller pods. | { }
```
